app/src/main/kotlin/com/yourssu/soomsil/usaint/screen/setting/SettingScreen.kt

```diff
@@ -228,11 +232,47 @@ fun SettingScreen(
     }
 }
 
+@Composable
+fun TitleText(
+    text : String
+) {
+    Box(
+        modifier = Modifier
+            .height(42.dp)
+            .padding(horizontal = 20.dp),
+        contentAlignment = Alignment.Center,
+    ) {
+        Text(
+            text = text,
+            fontWeight = FontWeight.Bold
+        )
+    }
+}
+
+@Composable
+fun ContentText(
+    text: String,
+    onClick: () -> Unit = {}
+) {
+    Box(
+        modifier = Modifier
+            .height(42.dp)
+            .padding(horizontal = 20.dp),
+        contentAlignment = Alignment.Center,
+    ) {
+        Text(
+            modifier = Modifier.clickable { onClick() },
+            text = text,
+            fontSize = 15.sp
+        )
+    }
+}
+
 @Preview
```

Comment:
```suggestion
@PreviewLightDark
```
이걸로 바꾸면 라이트모드, 다크모드 preview 모두 확인할 수 있어요